modules/jpg/image_loader_jpegd.cpp

```diff
@@ -35,6 +35,7 @@
 
 #include <jpgd.h>
 #include <jpge.h>
+
 #include <string.h>
```

Comment:
This might not be enforced very consistently yet, but for thirdparty includes I'm trying to also order them by how "local" they are. Here we're in the jpeg loader so jpeg includes come first as they're the most relevant for this code. Then general C/C++ std lib stuff or other system headers come afterwards.